### PR TITLE
Add support for macOS runners

### PR DESCRIPTION
**Pull request by evgeniiz321**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -84,6 +94,13 @@ runs:
       run: |
         echo "$NEOFS_WALLET" | base64 -d > "$GITHUB_ACTION_PATH/wallet.json"
 
+    - name: Install required package if running on macos 
+      shell: bash
+      run:   |
+        if [ "$RUNNER_OS" == "macOS" ]; then
```
[the remark this answers is not in the document]
Comment (evgeniiz321) [reply action.yml]: No reason, just a different way to do the same thing. Replaced with a conditional step.

File: action.yml [hunk 2]
```diff
@@ -56,14 +56,24 @@ outputs:
 runs:
   using: composite
   steps:
-    - name: Download latest stable neofs-cli for uploading reports to NeoFS
+    - name: Download latest stable neofs-cli for uploading reports to NeoFS for Linux
       uses: dsaltares/fetch-gh-release-asset@1.1.1
+      if: runner.os == 'Linux'
       with:
         repo: 'nspcc-dev/neofs-node'
         version: 'tags/v0.38.1'
```
Comment (roman-khimov) [inline action.yml:64]: Wow. Let's update it as well!

Comment (evgeniiz321) [reply action.yml]: done

**Comment by evgeniiz321** [comment]
For failing tests - https://github.com/nspcc-dev/neofs-node/issues/2850

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**